OpenIDConnectSwift/Info.plist

```diff
@@ -29,6 +18,17 @@
 	<string>1.0</string>
 	<key>CFBundleSignature</key>
 	<string>????</string>
+	<key>CFBundleURLTypes</key>
+	<array>
+		<dict>
+			<key>CFBundleTypeRole</key>
+			<string>Editor</string>
+			<key>CFBundleURLSchemes</key>
+			<array>
+				<string>com.okta</string>
```

Comment:
I would use a more unique name and actually a universal link domain would be even better for the example.